newdle/templates/base_email.html

```diff
@@ -0,0 +1,48 @@
+<table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+    style="margin-top: 20px; line-height: 1.7; color: #555;">
+    <tr>
+        <td>
+            <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+                style="max-width: 660px; font-family: Helvetica, Arial, sans-serif; font-size: 14px; background: #FFFFFF;">
+                <tr>
+                    <td style="border: 1px solid #ddd;">
+                        <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"
+                            style="border-collapse: collapse;">
```

Comment:
i'd align the attribute below the first attribute from the previous line (ie below `align`) or just not wrap it at all and keep the slightly longer lines